README.md

```diff
@@ -23,6 +23,7 @@ An Ansible role for gridinit. Specifically, the responsibilities of this role ar
 | `openio_gridinit_limits` | `dict` | Defines the max open files and limits for coredump |
 | `openio_gridinit_conf_location` | `{{ openio_gridinit_conf_confd }}/*/*` | Path of configurations to load |
 | `openio_gridinit_services` | `[]` | Defines services to configure |
+| `openio_gridinit_provision_only` | `false` | Toggle to handle reload manually |
```

Comment:
"services reload" maybe... I'm not fan of provision_only_mode, I'd choose something more descriptive of what is really happening `restart_services`, `avoid_service_restart` or something like that

Reply:
This tag is generic, in the long run it can be used to prevent other actions aswell, to ensure only provisioning is done